datacats/docker.py

```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```

Comment:
The problem there is that then we need some way to tell if the current command is remote-related.. 

I agree it's more pythonic though